globus-requirements.txt

```diff
@@ -0,0 +1,6 @@
+# extra requirements for globus
+-r ./requirements.txt
+requests>=2.0.0,<3.0.0
+six==1.10.0
+python-jose
+pycrypto
```

Comment:
`python-jose` and `pycrypto` are hidden dependencies. The sdk relies on them for some of the exchanges